install.sh

```diff
@@ -52,13 +52,18 @@ chmod 755 "$bin_dir/tiup"
 bold=$(tput bold 2>/dev/null)
 sgr0=$(tput sgr0 2>/dev/null)
 
-echo "Detected shell: ${bold}$SHELL${sgr0}"
-case $SHELL in
-    *bash*) PROFILE=$HOME/.bash_profile;;
-     *zsh*) PROFILE=$HOME/.zshrc;;
-         *) PROFILE=$HOME/.profile;;
-esac
-
+# Refrence: https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix
+shell=$(echo $SHELL | awk 'BEGIN {FS="/";} { print $NF }')
```

Comment:
We can add an integration test, which runs the `install.sh` scripts and uses `which tiup` to check the result.